src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
+  .menu-bars {
+    position: fixed;
+    left: calc(50% - 134px);
```

Comment:
This looks mysterious. Can you leave a comment here what the `50%-134px` are about?

Reply:
It's about positioning the menu at the center of the screen. You position the top left corner of the html-block at the center and go back 134px (half the width of the menu bar).